### PR TITLE
Python reference page

### PR DESCRIPTION
**Pull request by alex-tpom6oh**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝Changed routes:


| [**…n/reference/javascript**](https://docs-qsovmeg7g.avo-dev.app/implementation/reference/javascript) | [**…ation/reference/python**](https://docs-qsovmeg7g.avo-dev.app/implementation/reference/python) |
|:-:|:-:|
|<a href="https://docs-qsovmeg7g.avo-dev.app/implementation/reference/javascript">Screenshot of /implementation/reference/javascript</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/6DQ6Z1Y-hCf4F5SlijzCj07OhQpI3mkyInaaT7S2scnW8TTnUGYbrYHxQfDACt8Edi8FDD-XAJ-66NVfQvccSw.png">(view full size)</a> |<a href="https://docs-qsovmeg7g.avo-dev.app/implementation/reference/python">Screenshot of /implementation/reference/python</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/8koaxWBvVnbZUXKkXBVV4Iu-pT-RuSH9WovepiFARdZHlYAUTHAUQjptb2RW97RuMY9Yh3GB9HX8S-8_TdI_JA.png">(view full size)</a> |



| [**…-programming-languages**](https://docs-qsovmeg7g.avo-dev.app/implementation/supported-programming-languages) |
|:-:|
|<a href="https://docs-qsovmeg7g.avo-dev.app/implementation/supported-programming-languages">Screenshot of /implementation/supported-programming-languages</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/I8djQx9DPNcD-mqTfKEEIDb2gF3G3_ZKPOdTdmgpb1bMMRe8MmRA-cEwosPYRwtGOPg1O1O5RimEqw1NEh9f6Q.png">(view full size)</a> |







Commit 6e19aa9477a5987dc964899ebbd7595e18cf1b90 (https://docs-qsovmeg7g.avo-dev.app).

File: pages/implementation/reference/python.mdx [hunk 1]
```diff
@@ -0,0 +1,187 @@
+---
+layout: 'guide'
+title: 'Python'
```
Comment (logason) [inline pages/implementation/reference/python.mdx:3]:
```suggestion
title: 'Avo Functions in Python'
```

File: pages/implementation/reference/python.mdx [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+---
+
+import Link from '../../../components/Link';
+
+### Platforms
+
+Avo can generate Python code for your server side tracking.
```
Comment (logason) [inline pages/implementation/reference/python.mdx:10]:
```suggestion
Avo can code generate Avo Functions in Python for your server side tracking.
```

**Review by logason — APPROVED** [review]
LGTM!

**Pull request merged**